relayer/src/chain/cosmos.rs

```diff
@@ -200,10 +200,28 @@ impl CosmosSdkChain {
                 ));
             }
 
+            // Check on the configured max_tx_size against genesis block max_bytes parameter
+            let a = chain.rpc_client.genesis().await.map_err(|e| {
+                Error::health_check_json_rpc(
+                    chain_id.clone(),
+                    rpc_address.clone(),
+                    "/genesis".to_string(),
+                    e,
+                )
+            })?;
+            let max_allowed = a.consensus_params.block.max_bytes as f64 * 0.9;
```

Comment:
Nice work @adizere! 💯 Just curious about how we choose the `0.9` multiplier here. Also, might be nice to declare it as a const.

Reply:
Hehe good question! The `0.9` is just a heuristic, but arbitrary. I'm assuming if a transaction is larger than 90% of `max_bytes` then it may not necessarily fit in a block (because the block must accommodate a header as well). We could lower that threshold to 80% if we want to be extra cautious, but I can't imagine why it would deserve to be smaller than that.